package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "1.19.1",
+  "version": "1.19.2",
```

Comment:
Maybe make this a minor version update as we've changed the API.

Reply:
Fair, updated.

Reply:
Don't forget check-compatibility.js :-)